src/NServiceBus.Gateway/Settings/GatewaySettings.cs

```diff
@@ -155,7 +166,10 @@ public void AddReceiveChannel(string address, string type = "http", int maxConcu
 #if NET452
             var configSection = GetConfigSection(settings);
 
-
+            if (configSection?.TransactionTimeout != null)
+            {
+                logger.WarnFormat("The ability to specify transaction timeout via the GatewayConfig config section will be removed in v4. Instead use the `EndpointConfiguration.Gateway().TransactionTimeout(...)` API.");
```

Comment:
not sure tbh, but imho `Use the `EndpointConfiguration.Gateway().TransactionTimeout(...)` API instead.` reads better?